src/embedding/kd.ts

```diff
@@ -0,0 +1,441 @@
+// (C) 2020 GoodData Corporation
+import {
+    IGdcMessageEvent,
+    getEventType,
+    GdcProductName,
+    IGdcMessageEnvelope,
+    ISimpleDrillableItemsCommandBody
+} from './common';
+
+export namespace EmbeddedKpiDashboard {
+    /**
+     * Base type for KD events.
+     */
+    export type IKdMessageEvent<T, TBody> = IGdcMessageEvent<GdcProductName.KPI_DASHBOARD, T, TBody>;
+
+    /**
+     * Base type for KD event data.
+     */
+    export type IKdGdcMessageEnvelope<T, TBody> = IGdcMessageEnvelope<GdcProductName.KPI_DASHBOARD, T, TBody>;
```

Comment:
should we remove `Gdc`? So it will be consistent with `IKdMessageEvent`
`IKdGdcMessageEnvelope` => `IKdMessageEnvelope`

Reply:
updated